scripts/main.js

```diff
@@ -0,0 +1,79 @@
+const bookCard = document.getElementById('book-card')
+
+const myLibrary = []
+
+function Book(author, title, pages, read = false) {
+    this.id = Math.floor(Math.random() * 100)
+    this.author = author
+    this.title = title
+    this.pages = pages
+    this.read = read
+}
+
+function addBookToLibrary(author, title, pages, read) {
+    myLibrary.push(new Book(author, title, pages, read))
+    console.log(myLibrary)
```

Comment:
Let's remove the `console.log`.

> No `console.log` commited to repo. It's ugly, it kills performance. and it can make confidential data be visible in the browser tools. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/console_log_requirement.png) from Pathwright)